tools/eksDistroBuildToolingOpsTools/cmd/eksGoTool/cmd/createCveIssue.go

```diff
@@ -0,0 +1,147 @@
+
+			token, err := github.GetGithubToken()
+			if err != nil {
+				return fmt.Errorf("getting Github PAT from environment at variable %s: %v", github.PersonalAccessTokenEnvVar, err)
+			}
+			githubClient, err := github.NewClient(cmd.Context(), token)
+			if err != nil {
+				return fmt.Errorf("setting up Github client: %v", err)
+			}
+
+			// set up Issue Creator handler
+			issueManagerOpts := &issueManager.Opts{
+				SourceOwner: "rcrozean",
+				SourceRepo:  "eks-distro-build-tooling",
+			}
+			im := issueManager.New(retrier, githubClient, issueManagerOpts)
+
+			giOpts := &issueManager.GetIssueOpts{
+				Owner: "golang",
+				Repo:  "go",
+				Issue: upstreamIssueId,
+			}
+
+			upstreamIssue, err := im.GetIssue(cmd.Context(), giOpts)
+			if err != nil {
+				return fmt.Errorf("getting upstream issue: %v", err)
+			}
+
+			issueOpts := &issueManager.CreateIssueOpts{
+				Title:    GenerateIssueTitle(upstreamIssue),
+				Body:     GenerateIssueBody(upstreamIssue),
+				Labels:   &label,
+				Assignee: &assignee,
+				State:    &issueState,
+			}
+
+			if !dryRun {
+				_, err := im.CreateIssue(cmd.Context(), issueOpts)
+				if err != nil {
+					return fmt.Errorf("creating issue: %v", err)
+				}
+			}
+			return nil
+		},
+	}
+)
+
+func init() {
+	rootCmd.AddCommand(createCveIssue)
+	createCveIssue.Flags().StringVarP(&cveId, cveIdFlag, "c", "", "CVE ID")
+	createCveIssue.Flags().IntVarP(&upstreamIssueId, upstreamIssueIdFlag, "i", 0, "Upstream Issue ID e.g. ")
+	createCveIssue.Flags().StringVarP(&upstreamCommitHash, upstreamCommitHashFlag, "u", "", "Upstream Commit ID e.g. ")
+	createCveIssue.Flags().StringVar(&announcementSourceUrl, "announcemenSourceUrl", "", "Announcement Source URL e.g. https://groups.google.com/g/golang-announce/c/-hjNw559_tE/m/KlGTfid5CAAJ")
+	createCveIssue.Flags().BoolVar(&dryRun, "dry-run", true, "Output the results without opening github issues/prs")
+
+	requiredFlags := []string{
+		cveIdFlag,
+		upstreamIssueIdFlag,
+	}
+	for _, flag := range requiredFlags {
+		if err := createCveIssue.MarkFlagRequired(flag); err != nil {
+			log.Fatalf("failed to mark flag %v as requred: %v", flag, err)
+		}
+	}
+}
+
+func GenerateIssueBody(ui *gogithub.Issue) *string {
+	b := strings.Builder{}
+
+	if announcementSourceUrl != "" {
+		b.WriteString(fmt.Sprintf("From [Goland Security Announcemnt](%s)", announcementSourceUrl))
+	}
+
+	b.WriteString(fmt.Sprintf("For additional information for %s, go to the upstream issue %s", cveId, *ui.HTMLURL))
+
+	if upstreamCommitHash != "" {
+		b.WriteString(fmt.Sprintf(" and fix commit %s", upstreamCommitHash))
+	}
+
+	b.WriteString("\n\n")
+	b.WriteString(fmt.Sprintf("\n%s: %s", issueAutocreatedTemplate, authorNameFlag))
+	b.WriteString(fmt.Sprintf("\nTool Version: `%s`", version.Get().GitVersion))
+
+	bs := b.String()
+	if dryRun {
+		fmt.Printf("Created Issues Body: `%s`\n", bs)
```

Comment:
consider adding these outputs as debug level logs available with or w/out the dryRun flag